packages/graphql/lib/schema-builder/collections/field.directive.collection.ts

```diff
@@ -0,0 +1,18 @@
+import { MetadataListByNameCollection } from './metadata.list.by.name.collection';
+import { PropertyDirectiveMetadata } from '../metadata';
+
+export class FieldDirectiveCollection extends MetadataListByNameCollection<PropertyDirectiveMetadata> {
+  sdls = new Set<string>();
+  fieldNames = new Set<string>();
+
+  add(value: PropertyDirectiveMetadata) {
+    if (this.sdls.has(value.sdl) && this.fieldNames.has(value.fieldName))
+      return;
```

Comment:
```suggestion
    if (this.sdls.has(value.sdl) && this.fieldNames.has(value.fieldName)) {
      return;
    }
```
style: for the sake of consistency